src/main/java/org/dependencytrack/notification/publisher/AbstractWebhookPublisher.java

```diff
@@ -76,6 +76,8 @@ public void publish(final PublishContext ctx, final PebbleTemplate template, fin
             } else {
                 request.addHeader("Authorization", "Bearer " + credentials.password);
             }
+        } else if (getToken(config) != null) {
+            request.addHeader("X-Api-Key", getToken(config));
```

Comment:
Should the header name be configurable? I think the name will vary depending on what system the webhook is sent to.

Reply:
Yes, I will change it